themes/command_duration.theme.bash

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+if [ -z "$BASH_IT_COMMAND_DURATION" ] || [ "$BASH_IT_COMMAND_DURATION" != true ]; then
```

Comment:
please add this variable to `bash_profile.template.bash` so people will know to use it, maybe along with `COMMAND_DURATION_MIN_SECONDS`

Reply:
Fixed by https://github.com/Bash-it/bash-it/pull/1683/commits/9d3f28a8cc6ea58b92eae3474ef5191b2ab4921e